tests/test_python.py

```diff
@@ -44,7 +45,7 @@
         DerivedContainer(
             base=container_from_pytest_param(CONTAINER_T),
             containerfile=DOCKERF_PY_T1,
-            extra_launch_args=["-p", f"{port1}:{port1}"],
+            extra_launch_args=["-d", "-p", f"{port1}:{port1}"],
```

Comment:
Why is this necessary? `pytest_container` will launch your container in the background for you, you shouldn't have to add a deamonize flag to it.

Reply:
removed